Contributing.md

```diff
@@ -83,21 +84,27 @@ If you're reading this, you're probably creating a Pull Request or planning to d
 
 13. Create a PR to develop repository.
 
-# ERROR ❎ AND SOLUTION ✅ FOR COMMON PROBLEMS DURING SETUP 💻
+# ERROR ❎ AND SOLUTION ✅ FOR COMMON PROBLEMS DURING SETUP  FOR LOGIN ISSUES 💻
 
  1. Please remember to include an .env file in both the frontend and backend directories. 
  You can reference any two of the environment variables present in Discord for your configuration.
 
  2.Ensure Docker is running throughout the setup process. Make sure both the Redis stack and Progress 
  components are up and running.
+ 
+ 3. Right-click on homepage, inspect, navigate to console, access application > cookies, delete JWT files to troubleshoot login issues.
+
+ 4. Please use ESLint for code quality checks and Prettier for consistent code formatting before submitting pull requests.   This  ensures a clean and readable codebase. Thank you!
+
+# ERROR ❎ AND SOLUTION ✅ FOR COMMON PROBLEMS DURING SETUP  FOR BLOCKED PORT ERROR 💻
+   
```

Comment:
There is case where local postgress DB and redis might block the the dockerized postgres & redis , maybe  right a general solution to this